src/adapters/bidderFactory.js

```diff
@@ -235,6 +236,13 @@ export function newBidder(spec) {
         onBid: (bid) => {
           const bidRequest = bidRequestMap[bid.requestId];
           if (bidRequest) {
+            if (!bid.adapterCode) {
+              bid.adapterCode = adapterManager.aliasRegistry[bidRequest.bidder] || bidRequest.bidder;
```

Comment:
Unfortunately this is not enough, because you can alias an alias (this works: `pbjs.aliasBidAdapter('pubmatic', 'foobar'); pbjs.aliasBidAdapter('foobar', 'barfoo')`, and results in `aliasRegistry['barfoo'] === 'foobar'`)

Because of that I think you do need to walk up the tree as I outlined above. It's probably best to put it close to the aliasing logic, I suggest adding `getAdapterCode(bidderCode)` in `adapterManager.js`.

Reply:
Added a function `getAdapterCode` in `adapterManager.js`

Reply:
`getAdapterCode` function will traverse through the `aliasRegistry` to search for respective adapter. If return the `adapterCode` if it finds the entry in `aliasRegistry` or else it returns `null`